helloworld/greet/BUILD

```diff
@@ -1,28 +1,24 @@
 # Copyright 2020 Pants project contributors.
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-
 python_library(
-    # name defaults to the name of this directory, i.e., `greet`.
-    # sources defaults to ['*.py', '!*_test.py', '!test_*.py', '!conftest.py'].
+    # `name` defaults to the name of this directory, i.e., `greet`.
+    # `sources` defaults to ['*.py', '!*_test.py', '!test_*.py', '!conftest.py'].
```

Comment:
Should probably link to the docs we have on these defaults... https://www.pantsbuild.org/build_files.html#target-definitions or its replacement.